packages/list-store/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "list-store",
```

Comment:
Ah, I fought you did this for potentially sharing the packages and at this point, I was not planning to publish this package. 

The main reason I did not do this was because I didn't like the long names. I write all my terminal commands on the root level.  So if I need to install a new package, I would need to do `npm install newPackage -w @colonial-collections/list-store`. But this is properly not a good argument, so I will rename the package.